src/Model/Config/Source/Fields.php

```diff
@@ -32,6 +32,7 @@ public function getProductStockItemFields()
             ['value' => 'min_sale_qty', 'label' => __('stock_item -> min_sale_qty')],
             ['value' => 'max_sale_qty', 'label' => __('stock_item -> max_sale_qty')],
             ['value' => 'qty_increments', 'label' => __('stock_item -> qty_increments')],
+            ['value' => 'qty_backordered', 'label' => __('stock_item -> qty_backordered')],
```

Comment:
I'd change the label so it indicates it is part of the cart_item, not stock_item